test/extensions/unordered_list_with_checkboxes.unit

```diff
@@ -52,12 +44,12 @@
 *    [ ] four
 *     [ ] five
 <<<
+<p>*[ ] zero</p>
 <ul class="contains-task-list">
-<li class="task-list-item"><input type="checkbox"></input>​zero</li>
-<li class="task-list-item"><input type="checkbox"></input>​one</li>
-<li class="task-list-item"><input type="checkbox"></input>​two</li>
-<li class="task-list-item"><input type="checkbox"></input>​three</li>
-<li class="task-list-item"><input type="checkbox"></input>​four</li>
+<li class="task-list-item"><input type="checkbox"></input>one</li>
+<li class="task-list-item"><input type="checkbox"></input>two</li>
+<li class="task-list-item"><input type="checkbox"></input>three</li>
+<li class="task-list-item"><input type="checkbox"></input>four</li>
```

Comment:
@srawlins I do not know how could it happen. in your files there are two invisible whitespaces after each `</input>`

Reply:
Looks good